### PR TITLE
LPS-155785 Fix On Assignment options in Execution Type

### PR DESCRIPTION
**Pull request by igor-franca**
https://issues.liferay.com/browse/LPS-155785

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js [hunk 1]
```diff
@@ -37,18 +37,22 @@ const BaseActionsInfo = ({
 	setScript,
 	updateActionInfo,
 }) => {
+	if (
```
Comment (M-IgorFranca) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/shared-components/BaseActionsInfo.js:40]: And I also didn't understand why the onAssignment insertion does not occur in the rendering of the page when it is inside useEffect, we would have to fill in the previous fields in order to visualize the onAssignment.

**Comment by naraandrade** [comment]
@M-IgorFranca, I was taking a look at the flow in the `useEffect` of the `BaseActionsInfo` and the problem seems to be that, for a task, we are executing the code of the first `if` (ok, it is expected) but we are also entering the second `if` (not expected). So, we are putting the `On Assignment` option and just afterwards, we are removing it. I think you should check the condition for the second `if` to resolve this bug.

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js [hunk 2]
```diff
@@ -338,17 +338,6 @@ const NotificationsInfo = ({
 	};
 
 	if (selectedItem.type === 'task') {
-		if (
```
[the remark this answers is not in the document]
Comment (M-IgorFranca) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js]: sorry, that went unnoticed

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by M-IgorFranca** [comment]
I tried to update my branch and push again, but I didn't notice that the zulu repository was not synced with the upstream. Now I believe we need to close this PR and open another one, correct?

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**